### PR TITLE
Specify package versions in requirements.txt

### PR DESCRIPTION
**Pull request by andreepyro**
##### SUMMARY
Specifying package versions in the requirements file is not only a best practice but can prevent many future bugs.

##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-netaddr
-purefusion
+netaddr==0.8.0
```
Comment (sdodsley) [inline requirements.txt:1]: netaddr requirement is removed in #64

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+purefusion==1.0.10
```
Comment (sdodsley) [inline requirements.txt:2]: by pinning to a version you will require a new collection to be published each time a new purefusion module is released. 
Assuming the SDK is always going to be backwards compatible (which it should...) then there is no reason to pin to a version.

**Review by sdodsley — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (andreepyro) [reply requirements.txt:1]: You're right. But then, we should introduce upper bound, as backward compatibility is guaranteed only in minor and patch versions. Of course, that's the case only if we follow semantic versioning in `purefusion` module (it's not mentioned in Readme there, but I guess we do).
```suggestion
purefusion>=1.0,<2.0
```

**Review by sdodsley — APPROVED** [review]

**Comment by andreepyro** [comment]
@sdodsley I think you unintentionally didn't apply the suggestion and merger the PR with setting an exact package version `purefusion==1.0.10` If that's right, let me create a hotfix for that.

**Comment by sdodsley** [comment]
Yep - my bad - early in the morning

**Pull request merged**